File: C/313.c
#include <stdio.h>
#include <stdlib.h>

// 313 - Fin de Mes - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy
int main()
{
    int loop;
    scanf("%d", &loop);
    int s = 0;
    int c = 0;
    int i;
    for (i = 0;i<loop; i++) {
        scanf("%d", &s);
        scanf("%d", &c);
        if (s+c>=0) {
            printf("SI\n");
        } else {
            printf("NO\n");
        }
    }
    return 0;
}

File: Java/313.java
import java.util.Scanner;

/**
 *
 * 313 - Fin de Mes - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int bucle = sc.nextInt();
        for (int i = 1; i <= bucle; i++) {
            int saldo = sc.nextInt();
            int cambio = sc.nextInt();
            if (saldo + cambio >= 0) {
                System.out.println("SI");
            } else if (saldo + cambio < 0) {
                System.out.println("NO");
            }
        }
    }
}
